webpack.config.client.js

```diff
@@ -194,11 +194,17 @@ module.exports = ({
       new CopyWebpackPlugin({
         patterns: [{ from: 'public' }],
       }),
-      new DuplicatesPlugin({
+      new DuplicatePackageCheckerPlugin({
         // Emit compilation warning or error? (Default: `false`)
-        emitErrors: true,
+        emitError: true,
         // Display full duplicates information? (Default: `false`)
         verbose: true,
+        exclude({ name, path }) {
+          return (
+            name === '@babel/runtime' &&
+            path === './~/@emotion/react/~/@babel/runtime'
+          );
```

Comment:
The error message was:

```
  ERROR in @babel/runtime
    Multiple versions of @babel/runtime found:
      7.27.0 ./~/@emotion/react/~/@babel/runtime from ./~/@emotion/react/dist/emotion-react.browser.esm.js
      7.27.1 ./~/@babel/runtime from ./~/@babel/runtime/helpers/esm/defineProperty.js
```

We can just ignore the duplicate check for babel runtime in emotion